File: ALG_lista2/ex1.py
import math

num = [0.0]*10
soma_num = 0
soma_quadrado = 0

tamanho = len(num)

for i in range(tamanho):
    num[i] = float(input())
    soma_num += num[i]

media = soma_num / tamanho

for i in range(tamanho):
    diff = num[i] - media
    soma_quadrado += diff * diff

print(f"{math.sqrt(soma_quadrado / tamanho):.3f}")

File: ALG_lista2/ex1.c
#include <stdio.h>
#include <math.h>

int main(){

    float num[10];
    float soma_num = 0;
    float media;
    float soma_quadrado = 0;

    int tamanho = sizeof(num) / sizeof(num[0]);

    for(int i = 0; i < tamanho; i++){
        scanf("%f", &num[i]);
        soma_num += num[i];
    }

    media = soma_num / tamanho;

    for(int i = 0; i < tamanho; i++){
        float diff = num[i] - media;
        soma_quadrado += diff * diff;
    }

    float resultado = sqrtf(soma_quadrado / tamanho);
    printf("%.3f\n", resultado);

    return 0;
}